guides/doc-Managing_Hosts/master.adoc

```diff
@@ -3,11 +3,14 @@ include::common/header.adoc[]
 :context: managing-hosts
 :managing-hosts:
 
-// Render only for relevant and finished contexts
-ifdef::foreman-el,katello,satellite,orcharhino[]
-
 = {ManagingHostsDocTitle}
 
+// This guide is not ready for stable releases and not on Debian
```

Comment:
What's the reasoning here? Is it about [Chapter 2.4+](https://docs.theforeman.org/nightly/Managing_Hosts/index-foreman-el.html#Changing_a_Module_Stream_for_a_Host_managing-hosts) + Chapter 6 + Chapter 14?

If so, then maybe we can only hide specific assemblies?

Reply:
It's completely hidden today (https://docs.theforeman.org/nightly/Managing_Hosts/index-foreman-deb.html), which is what I went with. It's a good idea to hide specific assemblies and simplify it to `ifdef::HideDocument`, but I consider that out of scope for this PR.